src/main/java/org/spdx/sbom/gradle/project/ProjectInfo.java

```diff
@@ -49,6 +50,20 @@ static ProjectInfo from(Project project) {
         .build();
   }
 
+  static String version(Project project) {
```

Comment:
need to update `ProjectInfo.from` to use this, but I would prefer to re-arrange this so AppExtension (line 54) is calculated in the if(android) block. That might help us avoid explicitly shipping with the android build dependency?

Reply:
Making the android plugin a `compileOnly` dependency leads to a `ClassNotFoundException` even if we move the `AppExtension` code into the block that checks for the presence of the android plugin.